starter-code/src/functions-and-arrays.js

```diff
@@ -1,5 +1,17 @@
 // Find the maximum
 
+var num1;
+var num2;
+function maxOfTwoNumbers (num1, num2){
+  if(num1>num2){
+    console.log(num1);
+  } else if (num1<num2){
+    console.log(num2);
+  } else{
+    console.log('The numbers are equal!');
+  }
```

Comment:
This also can be written like:
```
  if (num1 > num2) {
+    console.log(num1);
+  } else {
+    console.log(num2);
+  }
```
If `num1 > num2` we show `num1`. Else means that `num2` is either greater or equal to `num1` . So we just can show `num2`